commands/issue.go

```diff
@@ -121,6 +121,12 @@ With no arguments, show a list of open issues.
 
 	-d, --since <DATE>
 		Display only issues updated on or after <DATE> in ISO 8601 format.
+
+	-o, --sort <SORT_KEY>
+		Sort displayed issues by "created" (default), "updated" or "comments".
+
+	-^ --sort-ascending
```

Comment:
this is for sorting, I thought about Sorted (s is taken), then sOrted (o is taken), then sort Ascending (a is taken), and then tried to thinkg outside of the box and thought about the little chevron shown in lists when you want to change ascending/descending sort. I'll take any better idea you can come up with.